tutorials/3d/procedural_geometry/arraymesh.rst

```diff
@@ -126,6 +158,42 @@ Put together the full code looks like:
         # Create mesh surface from mesh array.
         mesh.add_surface_from_arrays(Mesh.PRIMITIVE_TRIANGLES, arr) # No blendshapes or compression used.
 
+  .. code-tab:: csharp 
+
+    using Godot;
+    using System.Collections.Generic;
+
+    public class MeshInstance : Godot.MeshInstance {
+
+        public override void _Ready () {
+
+            Godot.Collections.Array arr = new Godot.Collections.Array ();
+            arr.Resize ((int)Mesh.ArrayType.Max);
+
+            var vertices = new List<Vector3> ();
+            var uvs = new List<Vector2> ();
+            var normals = new List<Vector3> ();
+            var indices = new List<int> ();
+
+            #######################################
+            ## Insert code here to generate mesh ##
+            #######################################
```

Comment:
Use C# comment syntax so that the resulting code is valid C#:

```suggestion
            ///////////////////////////////////////
            // Insert code here to generate mesh //
            ///////////////////////////////////////
```